### ISSUE TITLE
File support in 2.0

### ISSUE THREAD

**Issue by johansvenson** [potree/PotreeConverter#485]
I am sure it is stated somewhere and that I have simply missed it but it might be worth stating that the 2.0 version does not support the same file types as previous (e.g. ply file) ... At least I lost some time om figuring this out.

Thanks for an excellent product and I am really looking forward for the updated viewer!

**Comment by midnight-dev** [comment]
You're right. This has caused some confusion already, and the only place I think it's documented is within previous issues.

From what I remember, Markus focused on the LAS / LAZ format because they contain bounding box data, which is highly desirable for fast, parallel processing. Otherwise, the converter has to cope with the overhead of calculating bounding boxes manually as it discovers more points. If I understand the dilemma correctly, the converter would need to crawl through the entire point cloud before it starts the parallel indexing & chunking. There may be ways to speed that up, but if you already have a reliable bounding box via LAS, might as well use it. It also helps that there are plenty of utilities that support LAS files, like LASTools, libLAS, and CloudCompare.

If I don't see an update in a couple of days, I'll make a pull request with a brief note on compatibility.
